clientSrc/components/Registration.js

```diff
@@ -8,36 +8,38 @@ const Registration = () => {
     const [email, setEmail] = useState('');
 
     const handleChanges = (formItemBeingUpdated, e) => {
-        switch(formItemBeingUpdated){
-            case 'username':
-                setUsername(e.target.value)
-                break;
-            case 'password':
-                setPassword(e.target.value)
-                break;
-            case 'dateOfBirth':
-                setDateofBirth(e.target.value)
-                break;
-            case 'email':
-                setEmail(e.target.value)
+        switch (formItemBeingUpdated) {
+        case 'username':
+            setUsername(e.target.value);
+            break;
+        case 'password':
+            setPassword(e.target.value);
+            break;
+        case 'dateOfBirth':
+            setDateofBirth(e.target.value);
+            break;
+        case 'email':
+            setEmail(e.target.value);
+            break;
+        default:
         }
-    }
+    };
 
     const registerUser = (e) => {
```

Comment:
same thing as before, no need to comment console.logs

Reply:
Uncommented, see question above.